### DISCUSSION TITLE
OpenSCA-cli 3.1 feature wishlist

### DISCUSSION THREAD

**Discussion by cyberchen1995** [XmirrorSecurity/OpenSCA-cli#242] [Ideas]
Hi OpenSCA users & contributors,

OpenSCA-cli 3.0 was released over a mounts, and we are in the process of finalizing the plans for OpenSCA 3.1.0.

If you have any suggestions and ideas, feel free to open an [Feature request issue](https://github.com/XmirrorSecurity/OpenSCA-cli/labels/feat)

OpenSCA Team

**Comment by Ming825** [comment]
Hello, does OpenSCA support scanning of C++ code?
